components/input/input.css

```diff
@@ -0,0 +1,113 @@
+input{
+    outline: none;
+    border: none;
+    background: transparent;
+    font-size: 1.5rem;
+}
+
+.input-labeled{
+    position: relative;
+    font-size: 1.5rem;
+    padding: 7px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.input-labeled>label{
+    position: absolute;
+    top: -10px;
+    background: var(--background-color);
+    padding: 0 2px;
+}
+
+.outlined{
+    border: 2px solid var(--primary-color);
+    padding: 10px;
+    border-radius: 5px;
+    width: 25rem;
```

Comment:
Code is great! I liked the idea of removing the border of input tag and giving the border to its wrapper( div). It might be helpful in some cases